libs/shared/contentful/src/lib/queries/purchase-with-details-offering-content/util.ts

```diff
@@ -56,6 +58,7 @@ export class PurchaseWithDetailsOfferingContentUtil {
   }
 
   get purchaseCollection() {
-    return this.rawResult.purchaseCollection;
+    return this.rawResults.reduce((acc, curr) => ({ ...acc, ...curr }))
+      .purchaseCollection;
```

Comment:
I believe this won't work as expected. Using the spread operator like this will only spread the top level of the object. The nested objects will need to be spread out separately.